plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -149,6 +140,34 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 					);
 				}
 
+				// Handle both address_1 and address_2 fields together.
+				if ( field.key === 'address_1' ) {
+					return (
+						<AddressFields
+							id={ id }
+							addressType={ addressType }
+							key={ field.key }
+							address1={ addressFormFields.fields.find(
+								( customField ) =>
+									customField.key === 'address_1'
+							) }
+							address2={ addressFormFields.fields.find(
+								( customField ) =>
+									customField.key === 'address_2'
+							) }
+							addressFormFields={ addressFormFields }
+							fieldsRef={ fieldsRef }
+							onChange={ onChange }
+							values={ values }
```

Comment:
we don't pass values down to other props. What do you think about keeping this focused and only passing `address_1` and `address_2`?

Reply:
How would I retrieve the values of `values` in `<AddressFields/>` and `<Address2Field/>` then?

Reply:
I'm saying we should reconsider the use of `values` (Maybe it's not important and we can stick with what you have) but why are _all_ of the values needed? I see the `values` is used in the `onChange` in `Address2Field` but what if we only pass the _changed_ value (instead of spreading `values` as well), does that still work?

Reply:
Thanks for clarifying this, @opr. I see now what you mean. I just refactored
```tsx
// Handle both address_1 and address_2 fields together.
if ( field.key === 'address_1' ) {
	return (
		<AddressFields
			id={ id }
			addressType={ addressType }
			key={ field.key }
			address1={ addressFormFields.fields.find(
				( customField ) =>
					customField.key === 'address_1'
			) }
			address2={ addressFormFields.fields.find(
				( customField ) =>
					customField.key === 'address_2'
			) }
			addressFormFields={ addressFormFields }
			fieldsRef={ fieldsRef }
			onChange={ onChange }
			values={ values }
		/>
	);
}
```
to
```jsx
// Handle both address_1 and address_2 fields together.
if ( field.key === 'address_1' ) {
	const address1 = getFieldData(
		'address_1',
		addressFormFields.fields,
		values
	);
	const address2 = getFieldData(
		'address_2',
		addressFormFields.fields,
		values
	);

	return (
		<AddressFields
			address1={ address1 }
			address2={ address2 }
			addressType={ addressType }
			id={ id }
			key={ field.key }
			onChange={ onChange }
		/>
	);
}
```